packages/ui/src/views/repo/pull-request/components/labels/pull-request-labels-header.tsx

```diff
@@ -11,9 +12,18 @@ import {
   LabelValueType,
   TranslationStore
 } from '@/views'
+import { debounce } from 'lodash-es'
 
 import { LabelValueSelector } from './label-value-selector'
 
+const getSelectedValueId = (label?: LabelAssignmentType) => {
+  if (!!label && 'assigned_value' in label) {
+    return label.assigned_value?.id ?? undefined
+  }
+
+  return undefined
```

Comment:
No need to manually return `undefined`; if you don't return anything, you'll get the same
```suggestion
```